src/main/java/duke/command/EventCommand.java

```diff
@@ -0,0 +1,96 @@
+package duke.command;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Objects;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.task.Event;
+import duke.task.TaskList;
+import duke.ui.UI;
+
+/**
+ * Command to create an Event Task.
+ */
+public class EventCommand extends NonemptyArgumentCommand implements Command {
+
+    private static final String commandString = "event";
+    private final String arguments;
+
+    /**
+     * Constructor for an EventCommand
+     *
+     * @param arguments arguments for EventCommand
+     */
+    public EventCommand(String arguments) {
+        this.arguments = arguments;
+    }
+
+    /**
+     * Validate arguments for this command.
+     * They must,
+     * 1. Be in the format [description] /from YYYY-MM-DD /to YYYY-MM-DD
+     *
+     * @param arguments arguments to validate
+     * @throws DukeException if arguments are invalid
+     */
+    @Override
+    protected void validate(String arguments) throws DukeException {
```

Comment:
Maybe the exception message can include the full format of event commands for better clarity?